### ISSUE TITLE
Make ARP sign in page title more specific

### ISSUE THREAD

**Issue by darbyemadewell** [department-of-veterans-affairs/va.gov-team#138319]
## Severity

**Approximate Section 508 Severity**: Low

**VA Severity**: a11y-defect-2

## Behavior
**Expected**: 
The page title for `/representative/sign-in` should be descriptive of the content of the page and match or contain the content within the `h1`.

**Actual**: 
The page title for `/representative/sign-in` ("Accredited Represenative Portal | Veterans Affairs") is not descriptive of the content of the page and doesn't match the content within the `h1` ("Sign in or create an account").

## Examples

<img width="1624" height="1000" alt="Image" src="https://github.com/user-attachments/assets/c1e7d2d9-d3ef-4397-a3a3-b17b0e11c9e3" />
